docs/content/configuration-language.mdx

```diff
@@ -2,6 +2,7 @@
 title: Configuration Language
 sidebar_position: 2
 slug: /configuration-language
+description: Learn how to use configuration language to build a representation of a system's authorization model
```

Comment:
This seems a bit like an awkward sentence to me.

Can we make it: `Learn about OpenFGA's configuration language and using it to build a representation of a system's authorization model`?

Reply:
I try to avoid use of OpenFGA in the description so that other document that derives from this doc will not need to override the name.